wsgi_kerberos.py

```diff
@@ -102,10 +119,10 @@ class KerberosAuthMiddleware(object):
     :type auth_required_callback: callable
     :param read_max_on_auth_fail: When a request could not be authenticated,
         read and discard up to this many bytes of the request. This may help
-        naively- written clients that send large request bodies which they
+        naively-written clients that send large request bodies which they
         expect to be consumed before first confirming that the request was
         authenticated successfully. Pass 0 to disable this if you don't want to
-        waste resources to potentially accommodate such clients. Pass math.inf
+        waste resources to potentially accommodate such clients. Pass float('inf')
```

Comment:
Realized `math.inf` was added in Python 3, so updated this suggestion to be Python 2-compatible.